doc/api/cluster.markdown

```diff
@@ -357,18 +357,19 @@ This example will echo back all messages from the master:
       });
     }
 
-### worker.suicide
+### worker.exitedVoluntarily (formerly worker.suicide)
 
 * {Boolean}
 
 Set by calling `.kill()` or `.disconnect()`, until then it is `undefined`.
```

Comment:
Perhaps this should be done the same way as the other API migration?
i.e.:
```
### worker.suicide
Stability: 0 - Deprecated: Use worker.exitedVoluntarily instead.

An alias to `worker.exitedVoluntarily`.


### worker.exitedVoluntarily
…
```
Or something like that.